docs/ops/state/checkpoints.zh.md

```diff
@@ -26,69 +26,49 @@ under the License.
 * toc
 {:toc}
 
-## Overview
+## 概述
+checkpoint 使 Flink 的状态具有良好的容错性，通过 checkpoint 机制，Flink 可以对作业的状态和计算位置进行恢复。
```

Comment:
maybe the first "checkpoint" should be "Checkpoint"?

Reply:
@klion26 what do you mean is that at the begining of each paragrah is "Checkpoint" and other posititon is "checkpoint"?  thanks

Reply:
yes

Reply:
@klion26 I have updated all review suggestion,pls check ,thanks

Reply:
yes